include/boost/astronomy/coordinate/base_differential.hpp

```diff
@@ -115,6 +63,17 @@ struct base_differential
     {
         return this->diff;
     }
+    //! returns the differential of calling object
+    bg::model::point
+    <
+        CoordinateType,
+        DimensionCount,
+        CoordinateSystem
+    >
+    get_point() const
```

Comment:
Change the name of this function to `get_diff`

Reply:
By doing this we can extend all arithmetic function to work for differentials as well. Otherwise, we may get a conflict of .get_point() in all arithmetic functions.

Reply:
I guess this will help in `magnitude` function. But functions like `sum`, `mean`, `difference` use `make_cartesian_representation()` and hence can't be extended to work for differentials by defining `get_point` for `differential`.

Reply:
It'll also help to calculate cross for two differential classes or one representation class and one differential class. In the code where we used 
`bg::transform(cartesian1.get_point(), tempPoint1);
 bg::transform(cartesian2.get_point(), tempPoint2);
`

Reply:
@sarthak2007 I'm closing this PR. I have created a separate PR #127 for the extension of arithmetic functions to support differential systems.